apps/golden-sample-app/src/app/app.module.ts

```diff
@@ -10,6 +10,7 @@ import {
   EntitlementsModule,
   ENTITLEMENTS_CONFIG,
 } from '@backbase/foundation-ang/entitlements';
+import { AuthService, IdentityAuthModule } from '@backbase/identity-auth';
```

Comment:
What is the difference between this `AuthService` and `angular-oauth2-oidc` `OAuthService`?

Reply:
AuthService exposes an `isAuthenticated` observable which is used in the AuthGuard. The OidcAuthService handles refresh, logout, creating login urls etc.